### PR TITLE
Fix theme's title

### PR DESCRIPTION
**Pull request by ahuang11**
This should not be merged until bokeh's master is released because `from bokeh.themes import built_in_themes` doesn't exist yet in the latest stable version.

https://github.com/ioam/holoviews/issues/2871
image
image
image

### CODE REVIEW

File: holoviews/plotting/bokeh/element.py [hunk 1]
```diff
@@ -355,12 +356,24 @@ def _title_properties(self, key, plot, element):
...
-        opts = dict(text=title, text_color='black')
+        theme = self.renderer.theme
+        if theme is not None:
+            if theme in built_in_themes:  # built-in-theme name
+                opts = built_in_themes[theme]._json['attrs']['Title']
+            else:  # theme object
+                opts = theme._json['attrs']['Title']
+            opts['text'] = title
+        else:
+            opts = dict(text=title, text_color='black')
```
Comment (ahuang11) [inline holoviews/plotting/bokeh/element.py]: I feel like there's a way to simplify this logic, but my brain isn't working right now...

Comment (philippjfr) [reply holoviews/plotting/bokeh/element.py]: Since the default text_color is black anyway I think we can simply remove this bit entirely.

**Comment by ahuang11** [comment]
I feel like I tried to do that but the color wasn't black. I'll have to
test again.

**Comment by philippjfr** [comment]
You are right, the default color is ``'#444444'``. Not sure what to think, it won't be that noticeable a difference, but it's also not ideal.

**Comment by ahuang11** [comment]
It kind of was to me. Bold gray letters kind of is distracting especially
when everything else is black.

**Comment by philippjfr** [comment]
Another option would be to define a default theme, which defines a black title.

**Pull request merged**